documentation/user_docs/docs/manual/Reshaping_etc.rst

```diff
@@ -1,196 +1,282 @@
-#############
-Reshaping etc
-#############
+#####################
+Other shape functions
+#####################
 
-replicate
-=========
+``replicate``
+=============
 
-This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d, which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix is the same as the original one, or a d4d. In Matlab syntax this means e.g.
+This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d,
+which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix
```

Comment:
This is not clarifying for me. I thought that the d2d was best expressed as a 2D array of intensities over a 2D layer of image points. The replication to d3d would then stack multiple identical layers on top of each other in the relevant 3rd dimension. The use of matrices (array of matrices) suggests that the individual intensity values have become matrices.